src/MasterSite.php

```diff
@@ -1345,7 +1345,7 @@ private static function parse_youtube_url( string $url ): ?array {
 		$youtube_id = $matches[0][1] ?? null;
 
 		// For now just rel, but we can extract more from the url.
-		$query_string = 'rel=0';
+		$query_string = apply_filter( 'planet4_youtube_embed_parameters', 'rel=0' );
```

Comment:
@stduerre I wonder if `apply_filter` function did work on your local, because I assume you wanted to use `apply_filters` instead. 
Am I right?

Reply:
Ah yes of course. Thanks for the correction!